File: QuantConnect.AlphaStream.Python/Models/InsightPackage.py
from Models.Insight import Insight

class InsightPackage:
    """Package holding a group of insights emitted from one moment of time."""

    def __init__(self, json):

        self.AlphaId = json['alpha-id']

        self.AlgorithmId = json['algorithm-id']

        self.Insights = []

        insights = json.get('insights', None)
        if insights is None:
            return

        for i in insights:
            i['source'] = 'live trading'
            self.Insights.append(Insight(i))

File: QuantConnect.AlphaStream/Models/InsightPackage.cs
using Newtonsoft.Json;
using System.Collections.Generic;

namespace QuantConnect.AlphaStream.Models
{
    /// <summary>
    /// Wrapper packet to hold collection of Insight objects
    /// </summary>
    public class InsightPackage
    {
        /// <summary>
        /// Unique Alpha Id which generated this Insight
        /// </summary>
        [JsonProperty("alpha-id")]
        public string AlphaId { get; set; }

        /// <summary>
        /// Deploy id for this algorithm
        /// </summary>
        [JsonProperty("algorithm-id")]
        public string AlgorithmId { get; set; }

        /// <summary>
        /// Array of insights emitted at this timestep.
        /// </summary>
        [JsonProperty("insights")]
        public List<Insight> Insights { get; set; } = new List<Insight>();
    }
}